Libplanet/Tx/InvalidTxNonceException.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Libplanet.Blockchain;
+
+namespace Libplanet.Tx
+{
+    /// <summary>
+    /// The exception that is thrown when the <see cref="Transaction{T}.Nonce"/>
+    /// is different from <see cref="BlockChain{T}.GetNonce"/> result of
+    /// the <see cref="Transaction{T}.Signer"/>.
+    /// </summary>
+    [Serializable]
+    public sealed class InvalidTxNonceException : InvalidTxException
+    {
+        /// <summary>
+        /// Initializes a new instance of the
+        /// <see cref="InvalidTxNonceException"/> class.
+        /// </summary>
+        /// <param name="txid">The invalid <see cref="Transaction{T}"/>'s
+        /// <see cref="Transaction{T}.Id"/>.  It is automatically included to
+        /// the <see cref="Exception.Message"/> string.</param>
+        /// <param name="expectedNonce"><see cref="BlockChain{T}.GetNonce"/>
+        /// result of the <see cref="Transaction{T}.Signer"/>.</param>
+        /// <param name="improperNonce">The actual
+        /// <see cref="Transaction{T}.Nonce"/>.</param>
+        /// <param name="message">The message that describes the error.</param>
+        [SuppressMessage(
```

Comment:
This suppression doesn't seem necessary.

Reply:
It is necessary because of the following error:
```
InvalidTxNonceException.cs(34, 17): [SA1118] The parameter spans multiple lines
```
caused by https://github.com/planetarium/libplanet/pull/246/files#diff-9057bfb2853eef18bfabeec264493431R38